src/stores/sounds.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import { defineStore } from 'pinia'
+import Vue from 'vue'
+
+import { getCurrentUser } from '@nextcloud/auth'
+import { loadState } from '@nextcloud/initial-state'
+import { generateFilePath } from '@nextcloud/router'
+
+import BrowserStorage from '../services/BrowserStorage.js'
+import { setPlaySounds } from '../services/settingsService.js'
+
+/**
+ * Get play sounds option (from server for user or from browser storage for guest)
+ * @param {boolean} value whether sounds should be played
+ */
```

Comment:
```suggestion
/**
 * Get play sounds option (from server for user or from browser storage for guest)
 */
```
and did you mean to put it under `hasUserAccount` ?